crypto/rand/drbg_lib.c

```diff
@@ -1036,7 +1036,9 @@ static int drbg_bytes(unsigned char *out, int count)
     if (drbg == NULL)
         return 0;
 
+    rand_drbg_lock(drbg->parent);
```

Comment:
this looks wrong, drbg is thread local.
it does only rarely use the parent to reseed.
and it locks the parent before doing so.
Where was the original problem?

Reply:
this locks up the travis build, because the parent is locked again after several thousand iterations.

Reply:
Oww - crap. The change I did was end of november, When I finally managed to do wrestle my way through the procedure for committing a change I did a manual merge from my patches into the current openssl-clone. And promptly messed up :-(  Haste makes waste.
I'll have another look at that, and try to be quicker about it than last time,